lib/main/routes/app_routes.dart

```diff
@@ -1,18 +1,16 @@
 abstract class AppRoutes {
-  static const HOME = '/';
-  static const LOGIN = '/login';
-  static const MAILBOX = '$MAILBOX_DASHBOARD/mailbox';
-  static const MAILBOX_DASHBOARD = '/mailboxDashBoard';
-  static const THREAD = '$MAILBOX_DASHBOARD/thread';
-  static const EMAIL = '$THREAD/email';
-  static const SESSION = '/session';
-  static const COMPOSER = '/composer';
-  static const DESTINATION_PICKER = '/destinationPicker';
-  static const MAILBOX_CREATOR = '$MAILBOX_DASHBOARD/mailboxCreator';
-  static const MANAGE_ACCOUNT = '$MAILBOX_DASHBOARD/manage_account';
-  static const IDENTITY_CREATOR = '$MANAGE_ACCOUNT/identity_creator';
-  static const RULES_FILTER_CREATOR = '$MANAGE_ACCOUNT/rules_filter_creator';
-  static const EMAILS_FORWARD_CREATOR = '$MANAGE_ACCOUNT/emails_forward_creator';
-  static const SEARCH_EMAIL = '$MAILBOX_DASHBOARD/search_email';
-  static const CONTACT = '$MAILBOX_DASHBOARD/contact';
+  static const home = '/';
+  static const login = '/login';
+  static const session = '/session';
+  static const dashboard = '/dashBoard';
```

Comment:
`/dashboard`

Reply:
Oh, I have updated it in the last commits.